book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
```

Comment:
- [ ] Good job setting up the `book.rentals` in the Rental class. We can add a rental to the array with a Rental instance. But this is a `many-to-many` relationship so we should also be able to achieve the `has-many` association from the Book class itself also.

  Kindly create a method (can be named `add_rental`) that takes in a `date` and `person` as parameters and returns an instance of **`Rental`** with `date`, `self`(book) and `person` as its parameters.